src/utils.js

```diff
@@ -1,5 +1,68 @@
+import { CONVERSATION_STARTER, TemplateTypes } from './constants';
```

Comment:
This seems to be regressing. See https://github.com/twyla-ai/widget-core/commit/4b38728369916318a7b163d0e226126cf30d9806

Reply:
changed, but I had to keep some of the other code to return project name and workspace name which are used for the name of the cookie.